src/index.html

```diff
@@ -11,12 +11,115 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has long attribute values or the number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to the tag. Tag’s closing bracket should be on the same level as the opening one.